data/load-seed-data.js

```diff
@@ -21,16 +21,16 @@ async function run() {
         [user.email, user.hash]);
       })
     );
-      
-    const user = users[0].rows[0];
+    
+    // const  chord = chords[0].rows[0];
 
     await Promise.all(
-      animals.map(animal => {
+      chords.map(chord => {
         return client.query(`
-                    INSERT INTO animals (name, cool_factor, owner_id)
+                    INSERT INTO chords (chord, key, class, major, id)
                     VALUES ($1, $2, $3);
                 `,
-        [animal.name, animal.cool_factor, user.id]);
+        [chord.key, chord.chord, chord.id, chord.major, chord.class]);
```

Comment:
Really close here! The order that you `INSERT` your properties needs to match the order that you list your values in the square brackets here.

You also don't need to `INSERT` the id property, since SQL does this automatically on its own.